env/development/global.env

```diff
@@ -153,6 +153,9 @@ HERITABILITY_CONTAINER_VERSION=v0.04
 HERITABILITY_TASK_QUEUE_NAME=caendr-h2calc
 HERITABILITY_EXAMPLE_FILE=data/heritability_example_{SPECIES}.tsv
 
+PHENOTYPE_DB_ENABLED=True
```

Comment:
consider adding a comment with the expected format for the values (e.g.: literal python bool converted to string).